src/ipahealthcheck/core/main.py

```diff
@@ -13,14 +13,18 @@
 from ipahealthcheck.core.plugin import Result, Results
 from ipahealthcheck.core.output import output_registry
 from ipahealthcheck.core import constants
-from ipahealthcheck.meta.services import ServiceCheck
-
-from ipaserver.install.installutils import is_ipa_configured
-
 
 logging.basicConfig(format='%(message)s')
 logger = logging.getLogger()
 
+try:
+    from ipahealthcheck.meta.services import ServiceCheck
```

Comment:
I wonder if it would read better to have the try/except just around is_ipa_configured and move the ServiceCheck import after this block. The ServiceCheck import will fail because it has imporst of its own so the is_ipa_configured will probably never get hit but it provides the clearest view of what is being attempted IMHO.

Reply:
So I remember why I didn't do it at first: I provides a clearer view, but then it triggers flake8:
```
src/ipahealthcheck/core/main.py:26:1: E402 module level import not at top of file
```
I've added a noqa flag for this line.